src/security/test_security.py

```diff
@@ -4007,6 +4092,18 @@ def create_journals():
 
         return journal_one, journal_two
 
+    @staticmethod
+    def get_pii_strings_for_article(article):
+        pii_strings = []
+        for fa in article.frozen_authors():
+            pii_strings.append(fa.first_name)
+            pii_strings.append(fa.last_name)
+            pii_strings.append(fa.email)
+        pii_strings.append(article.correspondence_author.first_name)
+        pii_strings.append(article.correspondence_author.last_name)
+        pii_strings.append(article.correspondence_author.email)
+        return [string for string in pii_strings if string]
```

Comment:
To be thorough this should also test against department, institution, country, and orcid.

Reply:
I've added dept, inst and orcid but country is quite wide and could match language display so has been left out.

Reply:
OK, good call for this test. However, I do think country would be PII, since if an author is from a small country, the editor could use that to deduce who they are, when combining it with what they know about who is in their field.

Maybe what I'm looking for here is some documentation of which factors we are considering to be PII, so that we can refer to that when modifying these interfaces in the future. Could you add some documentation?

Reply:
I think this whole feature needs some docs so I’ll write it up tomorrow.